tests/unit_test/private/fed/app/simulator/simulator_runner_test.py

```diff
@@ -155,3 +157,24 @@ def test_start_server_app(self, mock_deploy, mock_admin, mock_register, mock_cel
 
                     runner.server.logger = Mock()
                     runner.server.engine.asked_to_stop = True
+
+    def test_get_new_sys_path_with_empty(self):
+        args = Namespace(workspace="/tmp")
+        args.set = []
+        runner = SimulatorClientRunner(args, [], None, None, None)
+        old_sys_path = copy.deepcopy(sys.path)
+        sys.path.insert(0, "")
+        sys.path.append("/temp/test")
+        new_sys_path = runner._get_new_sys_path()
```

Comment:
if we have "/temp/test", the new_sys_path should have this "/temp/test" ?

Reply:
The last entry needs to be removed.